inc/stat.class.php

```diff
@@ -1909,11 +1912,10 @@ private function generateCsvFile($labels, $series, $options = []) {
          ksort($values);
 
          if (!count($headers) && $options['title']) {
-            $headers[] = $options['title'];
+            $headers = $options['title'];
```

Comment:
In many cases, `$options['title']` is a string. it should not be changed.

Reply:
Can I create another option called $options['headers']  and look for it, if not present I take $options['title'] as done today ?

Reply:
Yes, it would be more appropriate.